src/Tribe/JSON_LD/Event.php

```diff
@@ -59,8 +59,18 @@ public function get_data( $posts = null, $args = array() ) {
 			$event_tz_string = get_post_meta( $post_id, '_EventTimezone', true );
 			$tz_mode         = tribe_get_option( 'tribe_events_timezone_mode', 'event' );
 			$tz_string       = $event_tz_string && $tz_mode === 'event' ? $event_tz_string : Tribe__Events__Timezones::wp_timezone_string();
-			$data->startDate = Tribe__Events__Timezones::to_utc( tribe_get_start_date( $post_id, true, Tribe__Date_Utils::DBDATETIMEFORMAT ), $tz_string, 'c' );
-			$data->endDate   = Tribe__Events__Timezones::to_utc( tribe_get_end_date( $post_id, true, Tribe__Date_Utils::DBDATETIMEFORMAT ), $tz_string, 'c' );
+
+			// Convert to local timezone
+			function local_timezone( $date, $event_tz_string ) {
```

Comment:
I love that this has been packaged up in a reusable way, but I strongly vote we do not define functions within functions and should instead move this out to a method of its own (it may also be worth looking to see if we have comparable existing functionality in one of our timezone classes).

Reply:
I moved it outside of the function...I did look but didn't see any other existing functions that do the same thing